portfolio/src/main/webapp/script.js

```diff
@@ -30,20 +49,64 @@ function fetchComments() {
  * @param {!Array<string>} comments 
  */
 function displayComments(comments) {
-  const commentList = document.getElementById("commentlist");
+  const commentList = document.getElementById("comment-list");
 
   //Remove all children of comment list to prevent dusplicate display
   while (commentList.firstChild) {
     commentList.removeChild(commentList.firstChild);
   }
 
   for (comment of comments) {
-    const commentListItem = document.createElement('li');
-    commentListItem.innerText = comment;
+    let displayName = comment["authorName"];
+    if (displayName === "") displayName =  comment["authorEmail"];
+
+    const commentListItem = createCommentItem(displayName, 
+      Number(comment["timestamp"]), comment["content"]);
     commentList.appendChild(commentListItem);
   }
 }
 
+/**
+ * Creates the DOM element for a comment
+ * @param {!string} author 
+ * @param {!number} timestamp
+ * @param {!string} content
+ */
+function createCommentItem(author, timestamp, content) {
+  const commentListItem = document.createElement('li');
+  const commentItemDiv = document.createElement('div');
+  const commentItemAuthor = document.createElement('h3');
+  const commentItemDate = document.createElement('p');
+  const commentItemTime = document.createElement('p');
+  const commentItemContent = document.createElement('p');
+
+  commentListItem.classList.add("comment-item");
+
+  commentItemAuthor.classList.add("comment-item-author")
+  commentItemDate.classList.add("comment-item-date")
+  commentItemTime.classList.add("comment-item-time")
+
+  const commentDate = new Date(timestamp);
+
+  commentItemAuthor.innerHTML = author;
+  commentItemDate.innerHTML = (commentDate.getMonth() + 1) + "/" 
+    + commentDate.getDate() + "/" 
+    + commentDate.getFullYear();
+  commentItemTime.innerHTML = commentDate.getHours() + ":" 
+    + commentDate.getMinutes() + ":"
+    + commentDate.getSeconds();
```

Comment:
When line wrapping, line break should come _after_ the operator.

https://google.github.io/styleguide/jsguide.html#formatting-where-to-break